data-processing/common/fetch_arxiv.py

```diff
@@ -16,6 +16,13 @@
 USER_AGENT = "Andrew Head, for academic research on dissemination of scientific insight <head.andrewm@gmail.com>"
 
 
+logger = logging.getLogger(__name__)
```

Comment:
nit pick: throughout out the code, most logging calls are the form of just `logging.func` (on the imported `logging` object) rather than `logger.func` on an initialized logger object. I am okay with using `logger` here instead. Is there an advantage?